config/routes.rb

```diff
@@ -317,11 +320,13 @@
       scope '/english-as-a-foreign-language' do
         get '/' => 'english_foreign_language/start#new', as: :english_foreign_language_start
         post '/' => 'english_foreign_language/start#create'
-        get '/proficiency/edit' => 'english_foreign_language/start#edit', as: :english_foreign_language_edit_start
-        patch '/proficiency/edit' => 'english_foreign_language/start#update'
+        get '/edit' => 'english_foreign_language/start#edit', as: :english_foreign_language_edit_start
+        patch '/edit' => 'english_foreign_language/start#update'
 
         get '/type' => 'english_foreign_language/type#new', as: :english_foreign_language_type
         post '/type' => 'english_foreign_language/type#create'
+        # get '/type/edit' ?
+        # patch '/type/edit' ?
```

Comment:
Are these 2 paths missing for a reason?

Reply:
mmm interesting - the controller does not appear to have an edit or update action. 
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/controllers/candidate_interface/english_foreign_language/type_controller.rb

@davidgisbey or @malcolmbaig can you shed some light on this?

Reply:
The TypeController doesn't create a resource, it just handles POSTs from the type form and decides which EFL controller to send you to next. Therefore, there isn't really a need for edit/update actions.

When a candidate chooses to "edit" their type, nothing actually happens to their saved record until they submit the subsequent EFL qualification form.